### PR TITLE
DOCS-637: crypto doc

### PR DESCRIPTION
**Pull request by q5sys**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Comment by bilditup1** [comment]
I suspect that because encryption operates at the dataset level, rather than the pool level, that Encryption should be mentioned under discrete subsections of 'ZFS Pools' (showing how to configure encryption during creation of nascent pools) and 'ZFS Datasets' (showing how to configure encryption during creation of nascent datasets as well as key management, locking and unlocking, etc) rather than making a new page under 'Configuring Storage', as is the case in this PR. Logically it would also convey that what is being encrypted is datasets and not pools, and this can be reinforced in the product itself by relabeling 'encryption' as 'encrypt root dataset' in the pool creation UI.

**Comment by q5sys** [comment]
I updated the intro paragraph to be clear about the root dataset and one other reference, the other references in the doc were all regarding the dataset.
I still feel that this being a single article under configure storage is a good idea as it gives a single location for information for dataset encryption that people are searching for. However I will add references to other documents to link back to this one if encryption is related.  

As to the UI labeling, I'd recommend creating a ticket here: https://jira.ixsystems.com/issues/
We will update the documentation if changes are put in place.

File: content/en/hub/initial-setup/storage/encryption.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+<br><br>
+
+The dataset listing changes to show the unlocked icon.
+
+## Conversion from GELI
+
+It is not possible to convert an existing FreeNAS/TrueNAS 11.3 or earlier GELI encrypted pool to use native ZFS encryption.
+However, data can be migrated from the GELI-encrypted pool to a new ZFS-encrypted pool.
+The new ZFS-encrypted pool must be at least the same size as the previous GELI-encrypted pool.
+Use `zfs send` to transfer the data to the ZFS-encrypted pool.
```
Comment (Mrt134) [inline content/en/hub/initial-setup/storage/encryption.md:108]: This last sentence begs the question "How do I use zfs send to transfer data"? Can we get more detail about how to do this? For example, something like "To transfer the data, open the shell in the web interface and use `zfs send`:
```
root@truenas <working example of zfs send to transfer data>
```

**Comment by q5sys** [comment]
@Mrt134 can you review this again?

**Pull request merged**